### DISCUSSION TITLE
`Copying from template version None` output when generating the template

### DISCUSSION THREAD

**Discussion by lnxpy** [copier-org/copier#1579] [Q&A]
I've created a copier template. Everything is alright with it. I'm generating it within a Python file. When prompting/answering is finished, it shows a message that I don't know how I can modify/fix.

```
Copying from template version None
 identical  .
 create ...
 create ...
```

What is "template version"?! What does "identical ." mean in here?!
My `copier.yml` only contains the questions and a post-gen message.

Thanks!

**Comment by lnxpy** [comment]
Hello @sisp. I'm just tagging you just in case you might've missed my issue.

**Comment by pawamoy** [comment]
Copier allows to [update projects](https://copier.readthedocs.io/en/stable/updating/) when a template evolves. To enjoy this feature, you have to use Git tags in your template repository. For example, tag `0.1.0` and tag `0.2.0`. If a project was generated with template v0.1.0, it can then be updated with changes from template v0.2.0, while keeping changes made in the project itself.

Here it says `version None`, probably because you don't have any tag in your template repository.

Lines that start with "identical" indicate that a file or folder hasn't change since last version.

**Reply by lnxpy** [reply]
Thank you for your response @pawamoy. The copier is part of my project and I'm using git tags, but still showing tha same message as Version is None for some reason!

How can I have  a tag version in my template repository?!

**Reply by pawamoy** [reply]
I'm not sure to understand. To help you further, we'd need you to open an issue with a reproducible example showing what's going wrong.

> How can I have a tag version in my template repository?!

[With a Git tag.](https://git-scm.com/book/en/v2/Git-Basics-Tagging)
